packages/webstudio-sdk/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": "../../node_modules/tsconfig/base.json",
```

Comment:
Please add the same code comment as here: https://github.com/webstudio-is/webstudio-designer/blob/46d0331671bd030f7a0519bf9d02bf59fa5491ea/apps/designer/tsconfig.json#L2